eng/pipelines/templates/Functional_Tests_On_Windows.yml

```diff
@@ -1,4 +1,12 @@
 steps:
+- task: PowerShell@1
+  displayName: "Set MSBuildEnableWorkloadResolver to false"
+  # Set MSBuildEnableWorkloadResolver to work around https://github.com/dotnet/sdk/issues/17461
+  inputs:
+    scriptType: "inlineScript"
+    inlineScript: |
+          Write-Host "##vso[task.setvariable variable=MSBuildEnableWorkloadResolver]false"
```

Comment:
Environment variables can be set more easily several ways:

* `variables` parameter on a job:

This sets the environment variable for all tasks in the job.

```yaml
jobs:
- job: jobName
  variables:
    MSBuildEnableWorkloadResolver: false
  steps:
```

* `env:` parameter on tasks

```yaml
- task: PowerShell@2
  filePath: path/to/script.ps1
  arguments: "-Arg value"
  env:
    MSBuildEnableWorkloadRes: false
```

* `variables:` at the pipeline definition

This makes the variable set for all stages, all jobs, all tasks. We already use it for something else: https://github.com/NuGet/NuGet.Client/blob/495a88382f5b357e62e83667a5f05192ee33e226/eng/pipelines/templates/pipeline.yml#L45-L46

Reply:
Thanks! Changed to use "variables parameter on a job".